### PR TITLE
Market decentralized/stranger80 resolver

### PR DESCRIPTION
**Pull request by tworec**

### CODE REVIEW

File: core/market/resolver/LICENSE [hunk 1]
```diff
@@ -1,21 +0,0 @@
...
-The above copyright notice and this permission notice shall be included in all
-copies or substantial portions of the Software.
-
-THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
-SOFTWARE.
```
Comment (nieznanysprawiciel) [inline core/market/resolver/LICENSE]: If we use this repo as subtree, we should probably leave license

Comment (tworec) [reply core/market/resolver/LICENSE]: @stranger80 can we use our current yagna license (GPL-3) for the resolver code also?

Comment (stranger80) [reply core/market/resolver/LICENSE]: Yes, sure.

Comment (tworec) [reply core/market/resolver/LICENSE]: thx :)

**Pull request merged**